tests/idea-space.spec.ts

```diff
@@ -63,8 +65,8 @@ test.describe('comment migration', () => {
 
     test("should update comment author", async () => {
         const strapi = await strapiConnect();
-        const params = { populate: ["user"] };
-        await strapi.db.query("api::comment.comment").update({ where: { author: user.username }, data: { user: null } });
+        const params = { where: { id: commentId }, populate: ["user"] };
```

Comment:
Needed to add a filter for specific comment by id, otherwise test fails sometimes on Postgres, as comments are also created in https://github.com/dev-launchers/strapiv4/blob/main/tests/event.spec.ts#L52